doc/docs.md

```diff
@@ -2130,6 +2103,82 @@ In this example, the `can_register` method has a receiver of type `User` named `
 The convention is not to use receiver names like `self` or `this`,
 but a short, preferably one letter long, name.
 
+### Embedded structs
+
+V support embedded structs .
+
+Basically, V embedding should work the same way as Golang embedding .
```

Comment:
The original starting comment was more clear.  The heading says what this section is talking about, so the first new sentence here is redundant.  The second sentence doesn't help anyone who is not familiar with Go embedding.

Reply:
removed